code/modules/genetics/mutations/spark_breath.dm

```diff
@@ -0,0 +1,46 @@
+//Macro: MUTATION_SPARK_BREATH
+//adds the ability to light cigs with your breath.
+/datum/genetics/mutation/spark_breath
+	name = "Spark Breath"
+	key = "MUTATION_SPARK_BREATH"
+	desc = "Allows the affected to breath a tiny flame from their maw."
+	gain_text = "You can breath the smallest of flames! Cute."
+	instability = 5 //literally is used to light cigs and smokables
+	exclusive_type = MUT_TYPE_MOUTH
+
+/datum/genetics/mutation/spark_breath/onMobImplant()
+	container.holder.verbs += /mob/living/proc/mutation_spark_breath
+
+
+/datum/genetics/mutation/spark_breath/onMobRemove()
+	container.holder.verbs -= /mob/living/proc/mutation_spark_breath
+
+/mob/living/proc/mutation_spark_breath()
+	set name = "Spark Breath"
+	set category = "Mutation"
+
+	if(stat!=CONSCIOUS)
+		usr.show_message("\blue You must be conscious to perform this ability!")
+		return
+
+	//safty check to remove the verb if the mutation isn't there.
+	var/datum/genetics/mutation/spark_breath/spk = src.unnatural_mutations.getMutation("MUTATION_SPARK_BREATH", TRUE)
+	if(!spk)
+		src.verbs -= /mob/living/proc/mutation_spark_breath
+		return
+
+	//Check if we have a smokeable in our hand.
+	if (istype(usr.get_active_hand(), /obj/item/clothing/mask/smokable))
+		var/obj/item/clothing/mask/smokable/ciggy = usr.get_active_hand()
+		usr.visible_message("[usr] lights \the [ciggy] with his breath.")
```

Comment:
```suggestion
		usr.visible_message("[usr] lights \the [ciggy] with their breath.")
```

Reply:
you can tell I was getting to the end of my coding push. Lots of spelling mistakes.